compiler/test/dotty/tools/vulpix/ParallelTesting.scala

```diff
@@ -788,14 +800,13 @@ trait ParallelTesting extends RunnerOrchestration { self =>
             if (actualErrors > 0)
               reporters.foreach(logReporterContents)
 
+            val checkFile = new JFile(dir.getAbsolutePath.reverse.dropWhile(_ == JFile.separatorChar).reverse + ".check")
```

Comment:
For`SeparateCompilationSource`, can we just use `new JFile(dir.getAbsolutePath + ".check")`?

Reply:
Yes